postgres/Create_Entity_User.sql

```diff
@@ -0,0 +1,22 @@
+-- Table: public.User
+
+-- DROP TABLE public."User";
+
+CREATE TABLE IF NOT EXISTS public."User"
```

Comment:
@FELIPEGRANADA next time you can take advantage of schemas to avoid creating tables in public schema